string surface_id
int64 x
int64 y
bool grasped
